### PR TITLE
Blackshield content!(and some marshal stuff too, ig.)

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
very long changelog, forgive me if i missed anything. <br>
adds *bleat complete with sound effects.<br>
Fixes the colt sprites to not be off center, thanks Kav!<br>
Adds junior officer jumpsuits to the wardrobe, and loadout..<br>
Moves turtlenecks from officer lockers to the wardrobe.<br>
Adds a full-body variant of the WO advanced plate carrier..<br>
New sprite for Marshal full-body plate carriers, courtesy of @Kavrick#5690 !.<br>
New full-helms for blackshield, available in standard+Flak armor kit., includes hooded variants in camo, tan, and grey.<br>
Also places one such helm in the COs locker.<br>
New bulletproof helmet for blackshield BP kit - no longer stuck with the standard ballistic helm..<br>
New corpsman full-helm included in Corpsman standard kit. Essentially the same as the other full-helm with slightly more bio protection.<br>
fixes the corpsman med-HUD goggles to ACTUALLY be HUDs and not just animated ballistics.<br>
New cloak for blackshield(or Marshals, i guess.) selectable in the loadout and with four variants, Grey, Green, Tan and Camo.<br>
At the request of @Fleshbound#3710 adds the option for Marshals to select a breaching hammer instead of a primary firearm..<br>
Also slightly nerfs said hammer to hit less often (2.0 delay ->2.2) - still hits like a truck, but significantly lower DPS than its stats might imply was intended to prevent it being used as a weapon..<br>
Sergeant locker now contains a ballistic gorget, essentially a unique looking helmet with some neat alts(goggles, cap, beret and "bane" variants)..<br>
Sergeant now also gets combat gloves like all the troopers do..<br>
Removes the uplink from the excel deferred crate at Dongels request.<br>
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
full-body variants for the shield vest in the tan and green are also in the works but if they're not ready i'll just PR 'em after. Till then they'll have their standard sprite.

**Comment by Dongels** [comment]
WOOOO! PR #3000!

File: code/modules/client/preference_setup/loadout/lists/factionsecurity.dm [hunk 1]
```diff
@@ -47,6 +47,14 @@
 	slot = slot_wear_suit
 	sort_category = "Faction: Security"
 
+datum/gear/factionsecurity/blackshield
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/factionsecurity.dm:50]:
```suggestion
/datum/gear/factionsecurity/blackshield
```

File: code/modules/clothing/accessories/accessory.dm [hunk 2]
```diff
@@ -294,6 +294,28 @@
...
+	icon_state = "blackshieldcloak_green"
+
+/obj/item/clothing/accessory/cape/blackshield/tan
+	name = "tan Blackshield cloak"
+	desc = "A simple, durable cloak for protecting you in any weather conditions! This one comes in a light tan."
+	icon_state = "blackshieldcloak_tan"
+
+/obj/item/clothing/accessory/cape/blackshield/camo
+	name = "camo Blackshield cloak"
+	desc = "A simple, durable cloak for protecting you in any weather conditions! This one comes in a tactical camo pattern"
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/accessories/accessory.dm:316]:
```suggestion
	desc = "A simple, durable cloak for protecting you in any weather conditions! This one comes in a tactical camo pattern."
```

File: code/modules/clothing/under/jobs/security.dm [hunk 3]
```diff
@@ -37,6 +37,16 @@
 	siemens_coefficient = 0.8
 	armor_list = list(melee = 5, bullet = 5, energy = 5, bomb = 0, bio = 0, rad = 0)
 
+/obj/item/clothing/under/rank/security/cadet
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/under/jobs/security.dm:40]: These two should be merged into 1 type that can be swapped

File: code/modules/clothing/head/helmet.dm [hunk 4]
```diff
@@ -78,6 +90,102 @@
...
+	item_state = "bs_fullhelm"
+	armor_list = list(melee = 30, bullet = 45,energy = 20, bomb = 30, bio = 0, rad = 0)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|BLOCKHAIR
+	body_parts_covered = HEAD|FACE|EARS
+	obscuration = LIGHT_OBSCURATION
+	action_button_name = "Toggle Headlamp"
+	brightness_on = 4
+	light_overlay = "bsfullhelm_light"
+
+/obj/item/clothing/head/helmet/ballistic/shieldfull/corpsman
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/head/helmet.dm:107]: This meant to be changeable into the base hoods?

File: code/game/machinery/vendor/weapon_kit_vendors.dm [hunk 5]
```diff
@@ -239,6 +241,7 @@
 /obj/machinery/vending/marshal_kit/proc/RedeemSpecPrimary(obj/item/voucher/voucher, mob/redeemer)
 	var/items = list(
 					"Thompson Kit" = /obj/item/storage/box/m_kit/thompson,
+					"Breacher-hammer Kit" = /obj/item/storage/box/m_kit/breacher,
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/weapon_kit_vendors.dm:244]: Why is this spec only?

File: code/modules/clothing/under/jobs/security.dm (hunk 3 above)
Comment (cdb-is-not-good) [reply code/modules/clothing/under/jobs/security.dm:40]: suuuuuuure.

File: code/modules/clothing/head/helmet.dm (hunk 4 above)
Comment (cdb-is-not-good) [reply code/modules/clothing/head/helmet.dm:107]: Nope, the bas hoods don't exist

File: code/game/machinery/vendor/weapon_kit_vendors.dm (hunk 5 above)
Comment (cdb-is-not-good) [reply code/game/machinery/vendor/weapon_kit_vendors.dm:244]: because i put it in the wrong voucher :)

**Comment by cdb-is-not-good** [comment]
should be all good to go now

**Pull request merged**